### PR TITLE
Apply border-radius onto linear gradient in bubble layout

### PR DESCRIPTION
**Pull request by t3chguy**
Fixes https://github.com/vector-im/element-web/issues/20534

---
This PR currently has no changelog labels, so will not be included in changelogs.

Add one of: `T-Deprecation`, `T-Enhancement`, `T-Defect`, `T-Task` to indicate what type of change this is plus `X-Breaking-Change` if it's a breaking change.

Preview: https://61e05101b1c23f006f94c91b--matrix-react-sdk.netlify.app
⚠️ Do you trust the author of this PR? Maybe this build will steal your keys or give you malware. Exercise caution. Use test accounts.

### CODE REVIEW

File: src/settings/enums/Layout.ts [hunk 1]
```diff
@@ -15,14 +15,9 @@ See the License for the specific language governing permissions and
...
-
 /* TODO: This should be later reworked into something more generic */
 export enum Layout {
     IRC = "irc",
     Group = "group",
     Bubble = "bubble",
 }
-
-/* We need this because multiple components are still using JavaScript */
-export const LayoutPropType = PropTypes.oneOf(Object.values(Layout));
```
Comment (andybalaam) [inline src/settings/enums/Layout.ts:28]: Did you mean to include this?

**Review by andybalaam — CHANGES_REQUESTED** [review]

**Comment by andybalaam** [comment]
LGTM otherwise.

File: src/settings/enums/Layout.ts (hunk 1 above)
Comment (t3chguy) [reply src/settings/enums/Layout.ts:28]: Yes, its a maintenance task that seems a little silly to have in its own PR. Bubbles related enough

**Review by andybalaam — APPROVED** [review]

File: src/settings/enums/Layout.ts (hunk 1 above)
Comment (andybalaam) [reply src/settings/enums/Layout.ts:28]: :+1:

**Pull request merged**